demo/map.js

```diff
@@ -2,6 +2,11 @@ var interactiveLayerId = 'label-amenity';
 var defaultType = defaultType || 'map';
 var cookieName = defaultType + 'Data';
 var popupPoi = null;
+var pMaxBounds;
```

Comment:
Jei neklystu šitų kintamųjų nėra būtina kurti. Pabandyk visus šiuos var ištrinti ir pažiūrėk ar viskas normaliai veikia.

Reply:
Jei neapibrėžti _map.js_ ir neapibrėžti kitur (tarkim _.html_'e), tai (Chrome) krenta klaida:
„Uncaught ReferenceError: pMaxBounds is not defined“.